### ISSUE TITLE
Ultima IV - Quest of the Avatar

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoScummVM#200]
I couldnt "Journey Onward", it would always tell me to Initialize a game first even though I had previously done so and had a save. To load a save game you actually need to press F7, but you cant do it from the menu as far as I can tell. Unless Ive missed something this means that you need to create a new character (not a quick process) and start a new game everytime you just want to load a previous game.

**Comment by Python-Exoproject** [comment]
Other points of note:

It is unclear if ScummVM "Ultima IV Enhanced" game ID is meant to be based of original game files or the files from a copy that has had the U4 Upgrade pack installed on it. For now I have used the original U4 game files as the U4 engine in ScummVM is based on xu4 project which required original files.

Im also noting here that it looks like all of the non EGA graphics sets use scalers that do smoothing instead of straight up 2x or 3x scale. Not everyone likes that look and it might be worth someone making a feature request to make this a choice.

**Comment by Python-Exoproject** [comment]
Journey Onward issue was caused by the fact we launch from command line. ScummVM adds some lines to the scummvm.ini to make it work which doesnt happen when using no ini file. I have set it up with our exoscummvm internal ini file and its now working. I have also confirmed that ive setup the enhanced version correctly, so only remaining point is that the non EGA graphics sets are all smoothed.
